src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -119,11 +147,22 @@ private SnowflakeSinkService getSink() {
    */
   @Override
   public void start(final Map<String, String> parsedConfig) {
-    long startTime = System.currentTimeMillis();
-    this.id = parsedConfig.getOrDefault(Utils.TASK_ID, "-1");
+    this.taskStartTime = System.currentTimeMillis();
+    long startTime = this.taskStartTime;
```

Comment:
why creating a local variable is needed here?

Reply:
it is unneeded, removed. thanks for the catch